source/access/mfa_login.rst

```diff
@@ -3,23 +3,219 @@
+Do not forget to set up your client so that it contacts your SSH agent
+when opening new connections (thereby making use of the certificates).
+For a few common clients the corresponding documentation pages are listed
+below.
+
+=========================================== ==================== =====================
+SSH Client name                             Purpose              Operating System
+=========================================== ==================== =====================
+:ref:`PuTTY <text mode access using PuTTY>` text-based terminal  Windows
+:ref:`MobaXterm <access using mobaxterm>`   text-based terminal  Windows
+:ref:`NoMachine <NX start guide>`           graphical desktop    Windows, Linux, MacOS
+:ref:`FileZilla <FileZilla>`                file transfer        Windows, Linux, MacOS
+=========================================== ==================== =====================
+
+Setting up an SSH agent for MFA
+-------------------------------
+
+The setup for your SSH agent depends on the choice of the operating system on your local machine,
+and that falls under either of the following two categories:
+
+-	Windows machines
+-	Linux or macOS machines 
+
+Windows machines
+~~~~~~~~~~~~~~~~
+
+Windows users are recommended to use Pageant as an SSH agent.
+For detailed information, please refer to the dedicated page about 
+:ref:`Using Pageant <using Pageant>`. 
+
+Linux and Mac machines
+~~~~~~~~~~~~~~~~~~~~~~
+
+On Linux and MacOS it is common to use OpenSSH, which is commonly integrated in the
+local operating system.
+For detailed information, please refer to :ref:`setting up your SSH agent for Linux and MacOS <SSH agent>`.
+Below, we provide recommendations on incorporating MFA in your new or existing SSH setup.
+
+- Before using your agent, it is best to verify the state of your agent.
+  Open a terminal. Verify if your agent is running with ``ssh-add -l``. 
+  If it is not running you will get the following error: 
+  ``Could not open a connection to your authentication agent``
+
+  If this is the case, start your agent with ``eval $(ssh-agent)``
+
+- You now need to create or adapt a profile for the cluster in your
+  ~/.shh/config file.
+  If you notice you do not have this file yet, create it
+  with the following command, you should adapt the permissions as well::
+
+     touch ~/.ssh/config
+     chmod 600 ~/.ssh/config
+   
+- Open your ``~/.ssh/config`` with a text editor, and make sure it looks like this::
+
+     Host login.hpc.kuleuven.be
+       ForwardAgent yes
+       PubkeyAuthentication yes
+       ChallengeResponseAuthentication yes
+       PreferredAuthentications publickey,keyboard-interactive
+        
+- You can now ``ssh`` to the cluster.
+  Upon the first successful connection, a so-called SSH certificate will be
+  produced and injected back into the SSH agent.
+  The SSH agent keeps the certificate while it runs silently in the background.
+  Bear in mind that the certificates are valid for maximum 16 hours.
```

Comment:
All of this can now be removed (as originally suggested in https://github.com/hpcleuven/VscDocumentation/pull/438#discussion_r1856930233):
```suggestion
```